server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/measure/Measure.java

```diff
@@ -349,22 +390,43 @@ public QualityGateStatus getQualityGateStatus() {
 
   /**
    * Any Measure, which ever is its value type, can have a Variations.
+   * @deprecated as only one period is now available. Use {@link #hasVariation()} instead
    */
+  @Deprecated
   public boolean hasVariations() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)